### PR TITLE
Add example on exploration to documentation.

### PR DESCRIPTION
**Pull request by kklein**

### CODE REVIEW

**Comment by kklein** [comment]
Kind bump @ivergara @YYYasin19

File: docs/source/examples/example_exploration.rst [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    )
+
+    req.add_column_subset_constraint()
+
+    columns1 = req[0].get_factual_value(engine)
+    columns2 = req[0].get_target_value(engine)
+
+    print(f"Columns present in both: {set(v1) & set(v2)}")
+    print(f"Columns present in only table1: {set(v1) - set(v2)}")
+    print(f"Columns present in only table2: {set(v2) - set(v1)}")
```
Comment (YYYasin19) [inline docs/source/examples/example_exploration.rst:212]: Do you mean 
```py3
print(f"Columns present in both: {set(columns1) & set(columns2)}")
# ...
```
here?

File: docs/source/examples/example_exploration.rst [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    Columns present in only table1: set()
+    Columns present in only table2: {'col_date'}
+
+
+Now, we can investigate the types of the columns present in both tables:
+
+
+.. code-block:: python
+
+    for column in set(v1) & set(v2):
```
Comment (YYYasin19) [inline docs/source/examples/example_exploration.rst:229]: The same here maybe?

**Review by YYYasin19 — COMMENTED** [review]
Sorry for the late reply, I already read it a few days ago but didn't have the time for a thorough review.

I just spotted a minor variable naming issue but otherwise the examples seem like a fine introduction on how to retrieve values using datajudge's internals. 🎉

File: docs/source/examples/example_exploration.rst (hunk 1 above)
Comment (kklein) [reply docs/source/examples/example_exploration.rst:212]: Yes, great catch!

File: docs/source/examples/example_exploration.rst (hunk 2 above)
Comment (kklein) [reply docs/source/examples/example_exploration.rst]: Yes!

**Review by ivergara — APPROVED** [review]
Good job!

**Comment by kklein** [comment]
Thanks @YYYasin19 and @ivergara !

**Pull request merged**